TestFoundation/TestHTTPCookieStorage.swift

```diff
@@ -246,6 +252,23 @@ class TestHTTPCookieStorage: XCTestCase {
         XCTAssertEqual(storage.cookies(for: url1!)!.count, 0)
     }
 
+    func descriptionCookie(with storageType: _StorageType) {
+        let storage = getStorage(for: storageType)
+        XCTAssertEqual(storage.description, "<NSHTTPCookieStorage cookies count:\(storage.cookies!.count)>")
```

Comment:
I know this is done elsewhere in the tests but I don't like force unwrapping. If it fails, the whole testsuite blows up.  It's better to do:

```swift
guard let cookies = storage.cookies else {
    XCTFail("No cookies")
    return
}
etc...
```